src/FormField/Input.php

```diff
@@ -40,6 +40,8 @@ class Input extends Generic
 
     public $actionLeft = null;
 
+    public $width = null;
```

Comment:
description of possible options and data type ?

Reply:
added.